types/spotify-web-playback-sdk/index.d.ts

```diff
@@ -0,0 +1,124 @@
+// Type definitions for spotify-web-playback-sdk 0.1
+// Project: https://beta.developer.spotify.com/documentation/web-playback-sdk/reference/
+// Definitions by: Festify Dev Team <mail@festify.rocks>
```

Comment:
This should be your github url, e.g.: `// Definitions by: My Self <https://github.com/me>`

>Error in spotify-web-playback-sdk
Command failed: node /home/travis/build/DefinitelyTyped/DefinitelyTyped/node_modules/dtslint/bin/index.js 
Error: /home/travis/build/DefinitelyTyped/DefinitelyTyped/types/spotify-web-playback-sdk/index.d.ts
ERROR: 3:37  dt-header  Error parsing header. Expected: /\<https\:\/\/github\.com\/([a-zA-Z\d\-]+)\>/. See: https://github.com/Microsoft/dtslint/blob/master/docs/dt-header.md
    at /home/travis/build/DefinitelyTyped/DefinitelyTyped/node_modules/dtslint/bin/index.js:74:19
    at Generator.next (<anonymous>)
    at fulfilled (/home/travis/build/DefinitelyTyped/DefinitelyTyped/node_modules/dtslint/bin/index.js:5:58)
    at <anonymous>